src/marine/smos_sss2ioda.py

```diff
@@ -99,7 +99,7 @@ def main():
 
     parser = argparse.ArgumentParser(
         description=(
-            'Read JPL/RSS SMOS sea surface salinity (SSS) file(s) and convert'
+            'Read JPL/RSS SMOS sea surface seaSurfaceSalinity (SSS) file(s) and convert'
```

Comment:
```suggestion
            'Read JPL/RSS SMOS sea surface salinity (SSS) file(s) and convert'
```

Reply:
@travissluka  Isn't this wording a bit excessive: ```sea surface seaSurfaceSalinity```?  Wasn't the old space separated wording just fine since this phrase is a description not a variable specifically?

Reply:
@gthompsnJCSDA @travissluka is suggesting what you are I think? The PR as-is changes it to `sea sureface seaSurfaceSalinity` probably because of a quick and dirty find/replace